kong/db/migrations/core/005_120_to_130.lua

```diff
@@ -1,3 +1,88 @@
+
+local function add_upstream_algorithm_to_postgress(connector)
```

Comment:
I don't see a reason for these to be local functions. It implies that the only teardown step for 1.2 to 1.3 migration is the addition of the `algorithm` field, and it could lead future migrations copy/pasted from this one to propagate this pattern. Better follow the pattern established in previous core migrations, and not name specific teardown steps in local functions.

Also, noting the (minor) typo: `_postgress`